model/src/test/kotlin/licenses/AbstractLicenseViewTest.kt

```diff
@@ -45,16 +45,30 @@ abstract class AbstractLicenseViewTest : WordSpec() {
 
                 view.getLicensesWithSources(packageWithoutLicense) should beEmpty()
 
+                view.getLicensesWithSources(packageWithConcludedLicense) should
```

Comment:
commit-msg: "Align the order **of the** packages.

Reply:
Done.